<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>桌面宠物</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(250, 250, 250);
	border:none
}
QComboBox{
	border:1px solid rgb(200,200,200);
	border-top-left-radius: 5px;
	border-top-right-radius:5px;
	background-color: rgb(255, 255, 255);
	color:rgb(0,0,0);
}
QComboBox::drop-down{
	border:0px;
}
QComboBox QAbstractItemView{
	border:none;
	background-color:rgb(255, 255, 255);
	color:rgb(0,0,0);
	selection-background-color: rgb(200, 200, 200);
	selection-color:rgb(255, 255, 255);
}
QComboBox QAbstractItemView:!enabled{
    color:rgb(125,125,125);
}
QSlider{
	border:none;
}
QSlider::groove:horizontal {
	border: 0px solid rgb(255,0,0);
	height: 6px;
}
QSlider::sub-page:horizontal {
	background-color: rgb(230, 230, 230);
}
QSlider::add-page:horizontal {
	background-color: rgb(250, 250, 250);
}
QSlider::handle:horizontal {
	margin: -7 0 -7 0;
	width: 18px;
	border-radius: 9px;
	background-color:rgb(0,129,255);
}
QScrollBar:vertical{
	border: none;
	width:8px;
	margin: 0 0 0 0;
	background: transparent;
}
QScrollBar::handle:vertical{
	background: rgb(200, 200, 200);
	border-radius:4px;
}
QScrollBar::handle:vertical:hover{
	background: rgb(160, 160, 160);
	border-radius:4px;
}
QScrollBar::handle:vertical:pressed{
	background: rgb(5, 160, 255);
	border-radius:4px;
}
QScrollBar::add-page,QScrollBar::sub-page:vertical{
	background: transparent;
}
QScrollBar::add-line,QScrollBar::sub-line:vertical{
	background: transparent;
}
QLineEdit{
	border:1px solid rgb(200,200,200);
	border-radius:5px;
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	selection-background-color: rgb(0, 0, 0);
	selection-color: rgb(255, 255, 255);
}
#savebutton,#helpButton{
	background-color: rgb(0, 180, 250);
	border-radius: 5px;
}
#savebutton:hover,#helpButton:hover{
	background-color: rgb(0, 200, 255);
	border-radius: 5px;
}
#showButton,#colorButton,#openButton,#newButton,#delButton,#delButton_2{
	border-radius:5px;
	color: rgb(0,0, 0);
	background-color: rgb(255, 255, 245);
}
#showButton:hover,#colorButton:hover,#openButton:hover,#newButton:hover,#delButton:hover,#delButton_2:hover{
	border-radius:5px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 252);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="8,1,1">
   <item row="10" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1,1,1">
      <item>
       <widget class="QLabel" name="copylabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>召唤宠物</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>召唤一个</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>收回一个</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>收回全部</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,2,1,2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="sizelabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>大小</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sizeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="opacitylabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>不透明度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="opacitySlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speedlabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>移动速度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="speedSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>40</pointsize>
      </font>
     </property>
     <property name="text">
      <string>控制面板</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="savebutton">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>保存设置</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1,1,1">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="modelabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>当前模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="modecomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>静默模式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>重力模式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>互动模式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>跟随鼠标</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>隐藏模式</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="objectlabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>互动道具</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="objectcomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>关闭</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>玩具</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>传送门</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="helpButton">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>帮助</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,1">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="switchfastlabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>外观切换键</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="switchcomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>空格</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F6</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="seedlabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>释放种子键</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="seedcomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>空格</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F6</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1,5,1">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="switchlabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>显示/隐藏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SwitchButton" name="switchButton" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="toplabel">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>窗口置顶</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SwitchButton" name="topButton" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1,1">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>背景颜色</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>Element.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
